catalog/data-model-extensions/@sine/unlocode/0.1.0/extension.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id": {
+    "namespace": "@sine",
+    "extension_name": "unlocode",
+    "version": "0.1.0"
+  },
+  "name": "UN/LOCODE Data Model Extension",
+  "files": ["schema.json"],
+  "author": {
+    "name": "SINE Foundation",
+    "email": "pact-catalog@sine.dev",
+    "url": "https://sine.foundation"
+  },
+  "license": "MIT",
+  "catalog_info": {
+    "summary": "The Pathfinder Framework Data Model allows for the interoperable exchange of product footprint data. This includes the product’s carbon footprint, represented in the Data Model as a CarbonFootprint. Each CarbonFootprint is scoped by Geography, specified (at maximum resolution) as a ISO 3166-2 country subdivision. In some cases, the current resolution is unsuficient and a more precise, hopefully plant-level, geographic specification would be recommended. The UN/LOCODE Data Model Extension aims at providing precisely the required level of granularity, using an international coding scheme that is public and can be integrated into solutions.",
```

Comment:
"unsuficient" -> "insufficient"

Reply:
Good catch! This must be fixed also in the documentation